test/test_package.py

```diff
@@ -3,6 +3,7 @@
 from torch.testing._internal.common_utils import TestCase, run_tests, IS_WINDOWS
 from tempfile import NamedTemporaryFile
 from torch.package import PackageExporter, PackageImporter
+from torch.package.module_environment import ModuleEnv, DefaultImporter
```

Comment:
If this is user facing it should be in `torch.package` and not just in the file it is defined in.